server/controllers/uncategorised.js

```diff
@@ -248,7 +276,9 @@ exports.listInExAccounts = function (req, res, next) {
 
   function process(accounts) {
```

Comment:
I also just add accounts Class 5, Patrick accounting consultant says
For some account for Class 1
   The loans can be budgeted, because they are considered a corporate asset that can not happen in an account  
   product (Class 7) but rather in the 16 
For some account for Class 2
  The acquisition of the assets may be budgeted because the accounting point of view this is not an expense 
   account that is used, rather an account of class 2

For some account for Class 5
  because of the accounting perspective, customer payments can be made by check or by payment order.
  This represents a product for the company.